recipes/ms-gsl/all/conanfile.py

```diff
@@ -1,16 +1,23 @@
 import os
 from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
 
-
-class GslMicrosoftConan(ConanFile):
+class MicrosoftGslConan(ConanFile):
     name = "ms-gsl"
     description = "Microsoft implementation of the Guidelines Support Library"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/microsoft/GSL"
-    topics = ("gsl", "header-only")
     license = "MIT"
+    topics = ("GSL", "guidelines", "core", "span")
```

Comment:
```suggestion
    topics = ("gsl", "guidelines", "core", "span")
```

Use lower case only. It's about a search engine, adding different cases can result in absent results.